packages/server/lib/modes/record.coffee

```diff
@@ -291,9 +303,9 @@ createRun = (options = {}) ->
     _.each response.warnings, (warning) ->
       switch warning.code
         when "FREE_PLAN_IN_GRACE_PERIOD_EXCEEDS_MONTHLY_PRIVATE_TESTS"
-          errors.warning("GRACE_PERIOD_WARNING", warning.daysLeft)
+          errors.warning("GRACE_PERIOD_WARNING", daysLeftPhrase(warning.daysLeft))
```

Comment:
so I will not add `daysLeft` just yet, so let's just use `used` and `limit` so projects in the grace period will show almost the same message as free plans

Reply:
aside from this - looks good, land it